client/views/omnichannel/directory/chats/contextualBar/RoomEdit.js

```diff
@@ -185,13 +189,18 @@ function RoomEdit({ room, visitor, reload, close }) {
 						<TextInput flexGrow={1} value={topic} onChange={handleTopic} />
 					</Field.Row>
 				</Field>
-				{Tags && (
+				{Tags && tagsList && tagsList.length > 0 ? (
 					<Field>
 						<Field.Label mb='x4'>{t('Tags')}</Field.Label>
 						<Field.Row>
 							<Tags value={Object.values(tags)} handler={handleTags} />
 						</Field.Row>
 					</Field>
+				) : (
+					<Field>
+						<Field.Label mb='x4'>{t('Tags')}</Field.Label>
+						<TagsManual tags={tags} handler={handleTags} />
```

Comment:
IMO we should have just one component, `Tags`, and deal with the different behaviors internally rather than importing multiple components in across the code base.